### PR TITLE
Add processor plugins

### PR DESCRIPTION
**Pull request by madhurimapoddar**
Ability to add processor plugins , eg. https://github.com/influxdata/telegraf/tree/master/plugins/processors/strings

### CODE REVIEW

**Review by LarryFinn — APPROVED** [review]

File: templates/telegraf.conf.j2 [hunk 1]
```diff
@@ -204,3 +204,18 @@
 ###############################################################################
 #                              service PLUGINS                                #
 ###############################################################################
+
+###############################################################################
+#                                  PROCESSORS                                    #
+###############################################################################
+
+{% if telegraf_processors is defined and telegraf_processors is iterable %}
```
Comment (MasterDDT) [inline templates/telegraf.conf.j2:212]: Nice, i didnt know there was an `is iterable`

File: templates/telegraf.conf.j2 [hunk 2]
```diff
@@ -204,3 +204,18 @@
...
+{% for item in telegraf_processors %}
+[[processors.{{ item.processor }}]]
+{% if item.config is defined and item.config is iterable %}
+{% for items in item.config %}
```
Comment (MasterDDT) [inline templates/telegraf.conf.j2]: `items` isnt a great iterator variable name =). How about `for itemConfig in item.configs`? and rename it to `configs`?

**Review by MasterDDT — APPROVED** [review]
Some comments

File: templates/telegraf.conf.j2 (hunk 2 above)
Comment (madhurimapoddar) [reply templates/telegraf.conf.j2]: so, its really 1 config per processor, so configs will confusing. Item here is a processor_plugin, and i will just call it that.

Comment (madhurimapoddar) [reply templates/telegraf.conf.j2]: i will change `item -> processor_plugin ` and
`items`-> `configs`

**Pull request merged**